geopandas/tests/test_sindex.py

```diff
@@ -1,15 +1,32 @@
 import sys
 
 from shapely.geometry import Point, Polygon
+from numpy.testing import assert_array_equal
 
 import geopandas
-from geopandas import GeoDataFrame, GeoSeries, base, read_file
+from geopandas import GeoDataFrame, GeoSeries, sindex, read_file
+from geopandas import _compat as compat
 
 import pytest
 
 
+@pytest.fixture()
+def pygeos_flag_fixture():
+    original_flag = compat.USE_PYGEOS
+    compat.set_use_pygeos(True)
+    yield
+    compat.set_use_pygeos(original_flag)
```

Comment:
CI already runs the geopandas tests twice: once with pygeos and once without. So is this then still needed?

Reply:
Agreed, I will try to clean this up, as well as the `get_pygeos` method. I'm sure there is a better way.

Reply:
Resolving, I removed this stuff.

This arose from wanting to create pygeos geometries to test with. My solution was to create shapely geometries (that can always be imported), combine them into a GeoSeries/GeoDataFrame and then index the geometry back out from `GeoDataFrame.geometry.values.data` so that the conversion is handled automatically.